apps/communications/manifest.webapp

```diff
@@ -117,7 +117,7 @@
                  "text/directory"]
       },
       "disposition": "inline",
-      "href": "/contacts/index.html?open",
+      "href": "/contacts/views/details/details.html",
```

Comment:
Remember! You need to duplicate the `open` activity, but with a different filter. This code is not ready yet to open a VCARD. This will be sorted in future bugs.